tests/managers_regress/models.py

```diff
@@ -147,3 +147,70 @@ class RelationModel(models.Model):
 
     def __str__(self):
         return force_text(self.pk)
+
+
+class BaseModel(models.Model):
```

Comment:
Since they don't need to be registered to the global `apps` I would use `isolate_apps` and defined them in the tests method.

The missing part #5613 tries to address is about models that must be registered to the global `apps` instance but are never queried against.